django-stubs/urls/resolvers.pyi

```diff
@@ -103,7 +103,7 @@ class URLPattern:
 
 class URLResolver:
     pattern: _Pattern
-    urlconf_name: str | None | Sequence[_AnyURL]
+    urlconf_name: str | None | Sequence[_AnyURL] | ModuleType
```

Comment:
Do we need to update the `__init__` type as well?

Reply:
Yes, it seems to be set in the `__init__` [here](https://github.com/django/django/blob/14917c9ae272f47d23401100faa6cefa8e1728bf/django/urls/conf.py#L74)